### ISSUE TITLE
[SCI] Benchmarking different architectures

### ISSUE THREAD

**Issue by chinglamchoi** [chandrikadeb7/Face-Mask-Detection#17]
For object detection, it would be interesting for participants to consider different models, e.g. YOLOv4 (https://arxiv.org/abs/2004.10934), YOLOv5 (https://github.com/ultralytics/yolov5), EfficientDet (https://arxiv.org/abs/1911.09070). Implementations of these models in Tensorflow, PyTorch and Darkent are readily available.
